changelog

```diff
@@ -1,3 +1,213 @@
+    * When multiple ships are selected in the outfitter and an outfit is also selected, an extra marker will appear next to any selected ships that have the selected outfit installed. (@TomGoodIdea)
+    * Replaced the mouse control toggle button with a preference to prevent accidental activations. (@samrocketman)
+    * Added a preference to toggle whether the flagship will collect flotsams. (@TomGoodIdea)
+    * Disabled ships now appear as gray in the escort UI at the bottom left to avoid confusion with docked ships, and selected ships have a pointer next to them. (@TomGoodIdea)
+    * The autofire preference can now be limited to only guns or only turrets. (@TomGoodIdea)
+    * Added a new control to auto-aim your ship toward your target when a key is pressed, default "E". Usable as an alternative to having the auto-aim setting always on. (@ChamEV, @UnorderedSigh)
+    * Only scan messages that are considered important will now be displayed. Important scan messages are when you are being scanned, or an escort with illegal cargo is being scanned. (@TomGoodIdea)
+    * Controls can now be unbound by pressing X or delete when hovering over the control. (@Koranir)
+  * Under the hood:
+    * Changed the Personality class from using a single integer to track personality flags to a bitset, allowing for more than just 32 NPC personality flags to exist. (@tibetiroka)
+    * The Personality class will fail compilation if the personality count enum doesn't match the number of possible personalities. (@UnorderedSigh)
+    * Corrected the declaration order of pirvate members of the Conversation class. (@warp-core)
+    * Added a missing #include to the Preferences class. (@quyykk)
+    * The planet, spaceport, bank, and trade panels are now defined by the interface file instead of entirely hardcoded, allowing for plugins to edit them. (@theweirdnut)
+    * Simplified various conversations using new "to display" nodes. (@warp-core)
+    * Cleaned up and updated information in the README and CONTRIBUTING files. (@quyykk)
+    * Cleaned up the syntax of the Ringworld Debris missions. (@warp-core)
+    * Refactored and improved the shop's handling of purchasing errors. (@alextd, @warp-core)
+    * The game will no longer print a warning if a GameAction uses a "fail" node for a mission name with no matching definition. (@Amazinite)
+    * Commands are now stored in a 64-bit integer, up from a 32-bit integer, to double the number of commands that can be used. (@UnorderedSigh)
+    * Moved the storage of save file information from your conditions to dedicated storage locations, such as licenses and the planets you have dominated. This information can still be accessed via derived conditions, using the same condition names that would have been used before (e.g. "license: <license name>"). (@petervdmeer)
+    * Added a missing copyright header to the tooltips file. (@mOctave)
+    * Created a cypher for generating Korath hails. (@UnorderedSigh)
```

Comment:
```suggestion
    * Created a cypher for generating Korath hails. (@UnorderedSigh and @ravenshining)
```

Reply:
The Korath language cypher is entirely @ravenshining's creation, which took her many hours of work to perfect. I just coded it up in python.

Reply:
Let it not be said that the protégé neglected to defend her absent sensai.